src/elements/common/error-boundary/ErrorBoundary.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import noop from 'lodash/noop';
+import { ERROR_CODE_UNEXPECTED_EXCEPTION, IS_ERROR_DISPLAYED } from '../../../constants';
+import DefaultError, { DefaultErrorProps } from './DefaultError';
```

Comment:
nit: can you swap this order?
```suggestion
import DefaultError, { DefaultErrorProps } from './DefaultError';
import { ERROR_CODE_UNEXPECTED_EXCEPTION, IS_ERROR_DISPLAYED } from '../../../constants';
```